### PR TITLE
add eqt

### PR DESCRIPTION
**Pull request by casperdcl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.
- [x] Once the PR is ready for review, please mention `@conda-forge/help-python, ready for review!` in a new comment.

### CODE REVIEW

**Comment by paskino** [comment]
I'm happy to be a maintainer

**Comment by casperdcl** [comment]
@conda-forge/help-python, ready for review!

File: recipes/eqt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - python >=3.7
+    - pyside2
+    - qdarkstyle
+
+test:
+  imports:
+    - eqt
+  commands:
+    - pip check
+    - env CI=1 pytest  # [not linux]
```
Comment (ocefpaf) [inline recipes/eqt/meta.yaml:33]: Are you planning on running the full test suit tests here? If not, you don't need this, the pytest dependencies below, and the source_files.

File: recipes/eqt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  requires:
+    - pip
+    - pytest
+    - pytest-cov
+    - pytest-timeout
+  source_files:
+    - test
+    - pyproject.toml
```
Comment (casperdcl) [inline recipes/eqt/meta.yaml:41]:
```suggestion
  requires:
    - pip
```

File: recipes/eqt/meta.yaml (hunk 1 above)
Comment (casperdcl) [reply recipes/eqt/meta.yaml]: not really needed as all tests are run upstream anyway. Removed :)

**Pull request merged**